doc/contributing/reconizing-contributors.md

```diff
@@ -20,6 +20,24 @@ Some of the benefits we hope to achieve through these programs include:
   done by individuals or teams like those mentioned. This should
   encourage other people to get involved and make similar contributions
 
+## Sponsorship link on nodejs/node README.md
+
+Collaborators may add a single link beside their name on the nodejs/node
```

Comment:
Is this only for collaborators, or is it for all organization members?

Reply:
@RedYetiDev, which non-collaborators listed on the README.md do you have in mind?

Reply:
None, I was just asking because the linked issue mentions more than just collaborators. I just wanted some clarification

Reply:
k thanks